bundles/org.openhab.binding.webthing/src/main/java/org/openhab/binding/webthing/internal/link/TypeMapping.java

```diff
@@ -48,70 +48,70 @@ public static ItemType toItemType(Property propertyMetadata) {
             case "MotionProperty":
             case "OnOffProperty":
             case "PushedProperty":
-                type = "switch";
+                type = "Switch";
                 tag = "Switchable";
                 break;
             case "CurrentProperty":
             case "FrequencyProperty":
             case "InstantaneousPowerProperty":
             case "VoltageProperty":
-                type = "number";
+                type = "Number";
```

Comment:
```suggestion
                type = CoreItemFactory.NUMBER;
```

Reply:
And so on... :-)

Reply:
Of course I can change here but as I checked all the bindings yesterday  I can tell you that most of them do not use these constants.